tests/safety/test_volkswagen_mqb.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env python3
```

Comment:
@jyoung8607 , you should simply rename the old file with `git mv`, so history is preserved. Seems like you deleted the old one and just made a new one.

Reply:
Hrm, I told PyCharm to rename it, and that's what came out. Let me see if I can unwind and reapply that commit more sensibly.